ci-operator/step-registry/baremetalds/capi/gather/baremetalds-capi-gather-commands.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# This script should collect all the logs generated previously as well
+# as collect a general state of the system under test
+echo "************ baremetalds capi gather command ************"
+
+if [[ ! -e "${SHARED_DIR}/server-ip" ]]; then
+  echo "No server IP found; skipping log gathering."
+  exit 0
+fi
+
+# Fetch packet basic configuration
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/packet-conf.sh"
+
+# Copy git source from current directory to the remote server
+tar -czf - . | ssh "${SSHOPTS[@]}" "root@${IP}" "cat > /root/source-code.tar.gz"
+
+### (mko) This function is a leftover from ci-operator/step-registry/baremetalds/assisted/gather/baremetalds-assisted-gather-commands.sh
+# function getlogs() {
+#   echo "### Downloading logs..."
+
+#   ssh  "${SSHOPTS[@]}" "root@${IP}" bash - <<EOF
+#   set -xeuo pipefail
+#   cd /home/assisted
+#   cp -v -R ./reports /tmp/artifacts || true
+#   find -name '*.log' -exec cp -v {} /tmp/artifacts \; || true
+# EOF
+
+#   scp -r "${SSHOPTS[@]}" "root@${IP}:/tmp/artifacts/*" "${ARTIFACT_DIR}"
+# }
+
+# Gather logs regardless of what happens after this
+# trap getlogs EXIT
+
+### (mko) This function is a leftover from ci-operator/step-registry/baremetalds/assisted/operator/gather/baremetalds-assisted-operator-gather-commands.sh
+# function getlogs() {
+#   echo "### Downloading logs..."
+#   scp -r "${SSHOPTS[@]}" "root@${IP}:/tmp/artifacts/*" "${ARTIFACT_DIR}"
+# }
```

Comment:
Why is it commented out?

Reply:
It's the discussion in the thread below - either we consume assisted-gather or we create capi-gather. If the former, we will just remove this completely; if the latter, we have a placeholder for the gather logic (which can't be just copy-pasted because e.g. `/home/assisted` does not exist)